dev/reference/sql/view.md

```diff
@@ -0,0 +1,65 @@
+---
+title: View
+summary: Learn how to use views in TiDB.
+category: reference
+---
+
+# View
```

Comment:
I'd go with the plural form.

Reply:
Haha, originally, I used `views`. Then I changed it after taking a look at the TOC.